fdbserver/MasterInterface.h

```diff
@@ -43,6 +44,7 @@ struct MasterInterface {
 	RequestStream<struct GetRawCommittedVersionRequest> getLiveCommittedVersion;
 	// Report a proxy's committed version.
 	RequestStream<struct ReportRawCommittedVersionRequest> reportLiveCommittedVersion;
+	RequestStream<struct GetTlogPrevCommitVersionRequest> getTlogPrevCommitVersion;
```

Comment:
Nit: I think `TLog` is used more often than `Tlog`.

Reply:
fixed